go/ops.go

```diff
@@ -91,25 +91,25 @@ func (op *InnerOp) CheckAgainstSpec(spec *ProofSpec) error {
 
 func prepareLeafData(hashOp HashOp, lengthOp LengthOp, data []byte) ([]byte, error) {
 	// TODO: lengthop before or after hash ???
-	hdata, err := doHashOrNoop(hashOp, data)
+	hdata, err := DoHashOrNoop(hashOp, data)
 	if err != nil {
 		return nil, err
 	}
 	ldata, err := doLengthOp(lengthOp, hdata)
 	return ldata, err
 }
 
-// doHashOrNoop will return the preimage untouched if hashOp == NONE,
+// DoHashOrNoop will return the preimage untouched if hashOp == NONE,
 // otherwise, perform doHash
-func doHashOrNoop(hashOp HashOp, preimage []byte) ([]byte, error) {
+func DoHashOrNoop(hashOp HashOp, preimage []byte) ([]byte, error) {
```

Comment:
If possible, I think it makes sense to keep this function private. Was there a reason to make it public?